app/Services/ForwardChatworkService.php

```diff
@@ -49,6 +49,9 @@ public function call()
                     if (is_bool($paramValue)) {
                         $paramValue = $paramValue ? 'true' : 'false';
                     }
+                    if (is_null($paramValue)) {
+                        $paramValue = "null";
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Services/ForwardChatworkService.php</br>**Line**: 53</br>String "null" does not require double quotes; use single quotes instead